doc/source/tutorial/stats/probability_distributions.rst

```diff
@@ -0,0 +1,447 @@
+Probability distributions
+-------------------------
+
+.. toctree::
+    :maxdepth: 1
+
+    continuous
+    discrete
+    building_distributions
+
+
+There are two general distribution classes that have been implemented
+for encapsulating :ref:`continuous random variables
+<continuous-random-variables>` and :ref:`discrete random variables
+<discrete-random-variables>`. Over 80 continuous random variables
+(RVs) and 10 discrete random variables have been implemented using
+these classes. Besides this, new routines and distributions can be
+easily added by the end user. (If you create one, please contribute it.)
```

Comment:
```suggestion
these classes. For mathematical reference information about individual
distributions, please see:

<links to continuous and discrete distribution reference pages>

Besides these, new distributions can be added by the end user.
```

Reply:
I captured this change but also added a ToC later in the document highlighting the distributions. Let me know what you think.